packages/next/server/next-dev-server.js

```diff
@@ -108,7 +108,7 @@ export default class DevServer extends Server {
       }
       if (!this.quiet) console.error(err)
     }
-
+    pathname = pathname === '/index' ? '/' : pathname
```

Comment:
I wonder if we should do this in render.tsx instead 🤔

Reply:
That works 👍